frontend/src/bundles/common/components/button/button.tsx

```diff
@@ -1,10 +1,44 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
 type Properties = {
-    label: string;
+    children: React.ReactNode;
     type?: 'button' | 'submit';
+    className?: string;
+    variant?: 'primary' | 'secondary' | 'plain' | 'round';
```

Comment:
Consider adding enum for such cases

Reply:
Done in commit [47118be](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/21/commits/47118be99a52c78ea88273915c6e45684e5815c4)